src/components/Links.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export function Links() {
+  const [openLinks, setOpenLinks] = useState(false)
+  const [createLink, setCreateLink] = useState(false)
+  const [name, setName] = useState('')
+  const [link, setLink] = useState('')
+  const [links, setLinks] = useState([])
+
+  function createLinkHandler() {
```

Comment:
use ES5 and later use ES6, maintain consistency

Reply:
yes will do it